web/client/actions/__tests__/login-test.js

```diff
@@ -0,0 +1,29 @@
+import expect from 'expect';
+import { onShowLogin } from '../login';
+import { SET_CONTROL_PROPERTY } from '../controls';
+import {isFunction} from 'lodash';
+
+
+describe('login actions', () => {
+    describe('onShowLogin', () => {
```

Comment:
is good to test only onShowLogin ?

Reply:
added unit tests for all the actions that can be tested